app/src/main/java/org/p2p/wallet/user/repository/UserInMemoryRepository.kt

```diff
@@ -131,7 +129,7 @@ class UserInMemoryRepository(
         val tokenData: TokenData? = findTokenData(mintAddress)
         return if (tokenData != null) {
             val price = getPriceByTokenId(tokenData.coingeckoId)
-            tokenConverter.fromNetwork(tokenData, price)
+            TokenConverter.fromNetwork(tokenData, price)
```

Comment:
any problems with the field?

Reply:
Yeah you inject this object, but forgot to provide it to module )

Reply:
it's was injected only in this class